File: programmers 문제 모음/연습문제/나누어떨어지는숫자배열.py
def solution(arr, divisor):
    answer = []
    for a in arr:
        if a % divisor == 0: # 나누어지면 answer에 저장
            answer.append(a)
    answer.sort() # 오름차순 정렬
    if len(answer) == 0: answer = [-1]
    return answer

File: programmers/연습문제/나누어떨어지는숫자배열.js
function solution(arr, divisor) {
    let answer = [];
    // 나누어 떨어지는 값 찾기
    for (let i = 0; i < arr.length; i ++) {
        if (arr[i] % divisor === 0) {
            answer.push(arr[i])
        }
    }
    // 오른차순으로 정렬하기
    answer.sort(function(a, b) {
        return a - b
    })
    // 빈배열일 조건 만족하기
    if (answer.length === 0) {
        answer = [-1]
    }
    return answer;
}